index.html

```diff
@@ -64,8 +74,133 @@ <h1>I’m Lilian<br>
           </li>
         </ul>
       </div>
+
     </section>
   </section>
+  <section class="section-grid">
+    <div id="item-0"><img src="/images/SnapshootPortfolio.png" alt="sreenshot portfolio" class="responsive-image">
+    </div>
+    <div id="item-1">
+      <p>Tonic</p>
+    </div>
+    <div id="item-2">
+      <div class="flex-container">
+        <div class="flex-items">
+          <p>CANOPY &nbsp;<img src="/images/Counter.png" alt="canopy"></p>
+        </div>
+        <div class="flex-items">
+          <p>Back End Dev &nbsp;<img src="/images/Counter.png" alt="backend"></p>
+        </div>
+        <div class="flex-items">
+          <p>2015</p>&nbsp;
+        </div>
+        <div class="flex-items"></div>
+      </div>
+    </div>
+    <div id="item-3">
+      <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+    </div>
+    <div id="item-4"><button class="button button1">html</button>
+    </div>
+    <div id="item-5"><button class="button button2">css</button></div>
+    <div id="item-6"><button class="button button3">javascript</button></div>
+    <div id="item-7"><button class="buttonproject">See Project</button></div>
```

Comment:
- [ ] The list of project technologies (e.g., "JavasScript", "HTML" and "CSS") should be a list `ul` and each of them a `li`
- [ ] Lets also check in the button outlines, they are not grey in color on the original page. 😄